File: spec/python/test_multiple_use.py
import unittest

from multiple_use import MultipleUse

class TestMultipleUse(unittest.TestCase):
    def test_multiple_use(self):
        with MultipleUse.from_file("src/position_abs.bin") as r:

            self.assertEqual(r.t1.first_use.value, 0x20)
            self.assertEqual(r.t2.second_use.value, 0x20)

File: spec/lua/test_multiple_use.lua
local luaunit = require("luaunit")

require("multiple_use")

TestMultipleUse = {}

function TestMultipleUse:test_multiple_use()
    local r = MultipleUse:from_file("src/position_abs.bin")

    luaunit.assertEquals(r.t1.first_use.value, 0x20)
    luaunit.assertEquals(r.t2.second_use.value, 0x20)
end
